### DISCUSSION TITLE
paddlespeech tts 语速问题

### DISCUSSION THREAD

**Discussion by josephthlee** [PaddlePaddle/PaddleSpeech#2463] [Q&A] [answered]
paddlespeech_client tts 可以透过 --speed 调整语速, 请问 paddlespeech  tts 如何调整语速？
感谢!

**Comment by yt605155624** [comment, marked as answer]
Duplicate of https://github.com/PaddlePaddle/PaddleSpeech/issues/2410 自己执行后处理

**Reply by josephthlee** [reply]
好的，看起来蛮清楚的，感谢！
